packages/yourloops/app/main-lobby.tsx

```diff
@@ -51,19 +52,10 @@ const LOGIN_PATH = '/login'
 const INTENDED_USE_PATH = '/intended-use'
 const PUBLIC_ROUTES = [LOGIN_PATH]
 const ALWAYS_ACCESSIBLE_ROUTES = [INTENDED_USE_PATH]
-const EXTERNAL_THEME_ROUTES = [NEW_CONSENT_PATH, RENEW_CONSENT_PATH, COMPLETE_SIGNUP_PATH, LOGIN_PATH, INTENDED_USE_PATH]
+export const ROUTES_REQUIRING_LANGUAGE_SELECTOR = [RENEW_CONSENT_PATH, NEW_CONSENT_PATH, TRAINING_PATH, COMPLETE_SIGNUP_PATH, INTENDED_USE_PATH, LOGIN_PATH]
```

Comment:
This variable is only used in the footer component, maybe it should be defined there?